### ISSUE TITLE
Missing field "price"

### ISSUE THREAD

**Issue by pineportal** [smartstore/SmartStoreNET#2096]
https://developers.google.com/search/docs/data-types/product

when product not in stock
![image](https://user-images.githubusercontent.com/31403448/94160605-a3cc2700-fe91-11ea-9c64-cac4f5c6b0b7.png)

**Comment by pineportal** [comment]
lowPrice and highPrice when product use attributes

```
 <div itemprop="offers" itemtype="http://schema.org/AggregateOffer" itemscope>
        <meta itemprop="lowPrice" content="119.99" />
        <meta itemprop="highPrice" content="199.99" />
        <meta itemprop="offerCount" content="6" />
        <meta itemprop="priceCurrency" content="USD" />
      </div>
```

**Comment by pineportal** [comment]
add itemCondition  schema

     DamagedCondition
    NewCondition
    RefurbishedCondition
    UsedCondition

**Comment by szokka** [comment]
Could not really reproduce missing field 'price' when not in stock, however, when for example the product is marked as call for price or show price after login, the meta property of price willl not be set.
ItemCondition had been implemented with #1116
